Client/Frontend/Shields/SimpleShieldsView.swift

```diff
@@ -59,20 +82,32 @@ class SimpleShieldsView: UIView, Themeable {
                     string: Strings.Shields.blockedCountLabel,
                     attributes: [.font: UIFont.systemFont(ofSize: 13.0)]
                 )
-                let attachment = ViewTextAttachment(view: self.infoButton)
-                string.append(NSAttributedString(attachment: attachment))
                 return string
             }()
             $0.backgroundColor = .clear
-            $0.setContentCompressionResistancePriority(UILayoutPriority(999), for: .horizontal)
+            $0.setContentCompressionResistancePriority(.required, for: .horizontal)
         }
         
         let infoButton = Button().then {
-            $0.setImage(UIImage(imageLiteralResourceName: "shields-help"), for: .normal)
-            $0.hitTestSlop = UIEdgeInsets(top: -10, left: -10, bottom: -10, right: -10)
+            $0.setImage(#imageLiteral(resourceName: "shields-help").withRenderingMode(.alwaysTemplate), for: .normal)
```

Comment:
can use `.template` instead of `.withRenderingMode(.alwaysTemplate)` we have a helper method for it